charts/flagsmith/templates/deployment-api.yaml

```diff
@@ -62,6 +62,7 @@ spec:
       initContainers:
 {{- if eq .Values.databaseExternal.enabled false }}
```

Comment:
This clause makes things slightly interesting, since the `waitfordb` management command can also wait for the migrations which could be useful for external databases. Maybe this is irrelevant for now since we're just using it to replicate the behaviour we had before.

Reply:
Does `manage.py waitfordb` wait for the migrations to run by default? If so, we would have a problem here, as we do:
- waitfordb
- migrate-db
- run web

Assuming it doesn't by default, then I think we're fine here and we just replicate previous behaviour.

However, sounds like for the task processor, we might want:
- waitfordb (with its `--and-also-for-migrations` flag set)
- run tasks

So perhaps scope for this PR is one of:
- maintain existing behaviour (which I _think_ is what we have atm on this PR)
- configure some new behaviour where we wait for external databases too, and do some waiting prior to starting the task processor

I think the second one is a sensible idea, just whether it should be separated from this PR.

Reply:
>Does manage.py waitfordb wait for the migrations to run by default?

It does not. See [here](https://github.com/Flagsmith/flagsmith/blob/main/api/core/management/commands/waitfordb.py#L27).

Agreed with the rest of the response. Let's separate from this PR.